R/outpack_metadata.R

```diff
@@ -1,3 +1,27 @@
+##' Read metadata for a particular id. You may want to use
```

Comment:
Could let users pass a query here too? Could just leave that for later and see if anyone asks for it, I suppose end users might not run this function very often

Reply:
yeah, I wondered about this - but a "query" here is pretty complicated because the interface includes a bunch of things (e.g., parameters, search options, etc)